{"name": "test_login_invalid_credentials", "status": "passed", "start": 1752539049664, "stop": 1752539050053, "uuid": "b3cbd2ef-da9a-48ca-b098-33c11b6e849b", "historyId": "03420e715c12f56e7e5c2808d1ca3a29", "testCaseId": "03420e715c12f56e7e5c2808d1ca3a29", "fullName": "tests.test_user_login.TestUserLogin#test_login_invalid_credentials", "labels": [{"name": "story", "value": "Логин с неверными данными"}, {"name": "feature", "value": "User Login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "hachipurri.local"}, {"name": "thread", "value": "1922-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}